lib/docs/scrapers/react_bootstrap.rb

```diff
@@ -3,7 +3,7 @@ class ReactBootstrap < UrlScraper
     self.name = 'React Bootstrap'
     self.slug = 'react_bootstrap'
     self.type = 'simple'
-    self.release = '1.5.0'
+    self.release = '2.7.2'
```

Comment:
```suggestion
    self.release = '2.7.4'
```
https://github.com/react-bootstrap/react-bootstrap/releases/tag/v2.7.4